README.md

```diff
@@ -196,6 +196,7 @@ Accounts          | usermod\_cmd           | Command string to modify a user's g
 Accounts          | gpasswd\_add\_cmd      | Command string to add a user to a group.
 Accounts          | gpasswd\_remove\_cmd   | Command string to remove a user from a group.
 Accounts          | groupadd\_cmd          | Command string to create a new group.
+Core              | cloud\_logging\_enabled| `false` disable cloud logging.
```

Comment:
it seems the default config is setting it to true, isn't it?

Reply:
yes, but entries here do not define default values. Like `accounts_daemon` is [true](https://github.com/GoogleCloudPlatform/guest-agent/blob/main/google_guest_agent/cfg/cfg.go#L55) by default

Reply:
indeed.